deel-2-flexbox-layout/opdracht-3/styles/styles.css

```diff
@@ -1 +1,52 @@
-/*Schrijf hier jouw CSS*/
\ No newline at end of file
+/* -- Flex-row on Body class = flex-row -- */
+
+.flex-row {
+  display: flex;
+  min-height: 100vh;
+}
+
+
+
+.flex-row-nav {
+  flex-basis: 250px;
+  flex-grow: 0;
+  flex-shrink: 0;
+}
+
+.flex-row-main {
+  flex-grow: 1;
+}
+
+/* -- Flex-column on div col-2 -- */
+
+.flex-column-layout {
```

Comment:
ik heb gezien dat jij een class met twee namen hebt gemaakt. ik heb dat gewoon met een naam gemaakt en zelf de properties daarin gezet en resultaten waren zelfde. Voor mij het is nieuw, is het voor meer duidelijkheid zo kan je elke stap duidelijk kan zien?

Reply:
Goede opmerking, Ik merk nu ook dat het soms niet nodig is geweest om een 2e class aan te maken, 

Maar ik pas dit inderdaad toe om onderscheid te maken. Dan weet ik wanneer ik bv de layout aanpas en daarna de row zelf. Op deze manier kan je ook onderscheid maken tussen styling (typo-kleurgebruik etc.) en display Flex. Dan weet je wat er op elk element aan Flex properties staan.

Mijn valkuil is alleen dat het soms teveel wordt en dit moet ik voor mezelf extra aanscherpen.